web/src/components/query-summary/query-summary.tsx

```diff
@@ -57,7 +57,7 @@ export const QuerySummaryContent: React.FC<{
         <Tooltip content={<Text component={TextVariants.p}>{t('Filtered sum of packets')}</Text>}>
           <Text id="packetsCount" component={TextVariants.p}>
             {t('{{count}} packets', {
-              count: flows.map(f => f.fields.Packets).reduce((a, b) => a + b)
+              count: _.isEmpty(flows) ? 0 : flows.map(f => f.fields.Packets).reduce((a, b) => a + b)
```

Comment:
that's fine, however writing `reduce((a, b) => a + b, 0)` should do the same and is much more idiomatic javascript. But lgtm

Reply:
I'm not used to `reduce` function. Let's move to this then. 
Thanks @jotak